cypress/e2e/tests/pages/users-and-auth/roles.spec.ts

```diff
@@ -263,102 +272,101 @@ describe('Roles Templates', { tags: ['@usersAndAuths', '@adminUser'] }, () => {
-        rolesList.resourceTable().sortableTable().pagination()
+        rolesList.resourceTable().sortableTable().paginationGlobalUsersRoles()
           .leftButton()
           .click();
 
         // check text and buttons after navigation
-        rolesList.resourceTable().sortableTable().pagination()
+        rolesList.resourceTable().sortableTable().paginationGlobalUsersRoles()
           .paginationText()
           .then((el) => {
-            expect(el.trim()).to.eq(`1 - 100 of ${ count } GlobalRoles`);
+            expect(el.trim()).to.eq(`1 - 10 of ${ count } GlobalRoles`);
           });
-        rolesList.resourceTable().sortableTable().pagination()
+        rolesList.resourceTable().sortableTable().paginationGlobalUsersRoles()
           .beginningButton()
           .isDisabled();
-        rolesList.resourceTable().sortableTable().pagination()
+        rolesList.resourceTable().sortableTable().paginationGlobalUsersRoles()
           .leftButton()
           .isDisabled();
 
         // navigate to last page - end button
-        rolesList.resourceTable().sortableTable().pagination()
+        rolesList.resourceTable().sortableTable().paginationGlobalUsersRoles()
           .endButton()
           .click();
 
-        // check row count on last page
-        rolesList.resourceTable().sortableTable().checkRowCount(false, count - 100);
-
         // check text after navigation
-        rolesList.resourceTable().sortableTable().pagination()
+        rolesList.resourceTable().sortableTable().paginationGlobalUsersRoles()
           .paginationText()
           .then((el) => {
-            expect(el.trim()).to.eq(`101 - ${ count } of ${ count } GlobalRoles`);
+            expect(el.trim()).to.eq(`${ count - (count % 10) + 1 } - ${ count } of ${ count } GlobalRoles`);
           });
 
         // navigate to first page - beginning button
-        rolesList.resourceTable().sortableTable().pagination()
+        rolesList.resourceTable().sortableTable().paginationGlobalUsersRoles()
           .beginningButton()
           .click();
 
         // check text and buttons after navigation
-        rolesList.resourceTable().sortableTable().pagination()
+        rolesList.resourceTable().sortableTable().paginationGlobalUsersRoles()
           .paginationText()
           .then((el) => {
-            expect(el.trim()).to.eq(`1 - 100 of ${ count } GlobalRoles`);
+            expect(el.trim()).to.eq(`1 - 10 of ${ count } GlobalRoles`);
           });
-        rolesList.resourceTable().sortableTable().pagination()
+        rolesList.resourceTable().sortableTable().paginationGlobalUsersRoles()
           .beginningButton()
           .isDisabled();
-        rolesList.resourceTable().sortableTable().pagination()
+        rolesList.resourceTable().sortableTable().paginationGlobalUsersRoles()
           .leftButton()
           .isDisabled();
       });
     });
 
     it('filter global roles', () => {
+      usersPo.goTo();
```

Comment:
not need since we have the `navTo`

Reply:
Added in accordance to: https://github.com/rancher/dashboard/blob/56f7893b89e1251a42ba1bf9925f8df6308721ca/cypress/e2e/tests/pages/users-and-auth/roles.spec.ts#L261

Reply:
oh i see